### PR TITLE
Added units explicitly in STIR documentation

### PR DESCRIPTION
**Pull request by skarrea**
Removes ambiguity between quantities expected in physical units
and quantities given in number of voxels.
Related issues: #1022 #826 
[ci skip]
[travis skip]

### CODE REVIEW

File: src/xSTIR/pSTIR/STIR.py [hunk 1]
```diff
@@ -235,36 +235,36 @@ def __del__(self):
...
         return parms.float_par(self.handle, self.name, 'length')
 
     def set_radius_x(self, value):
-        """Sets x radius."""
+        """Sets x radius in mm."""
         parms.set_float_par(self.handle, self.name, 'radius_x', value)
 
     def get_radius_x(self):
-        """Returns x radius."""
+        """Returns x radius. in mm"""
```
Comment (robbietuk) [inline src/xSTIR/pSTIR/STIR.py:250]:
```suggestion
        """Returns x radius in mm."""
```

**Review by robbietuk — CHANGES_REQUESTED** [review]
Looks good! Small gramatical correction.

File: src/xSTIR/pSTIR/STIR.py (hunk 1 above)
Comment (skarrea) [reply src/xSTIR/pSTIR/STIR.py]: Thanks, good catch!

**Review by KrisThielemans — APPROVED** [review]
looks good. all done? I think we can squash-merge this

**Comment by skarrea** [comment]
Yes, all done. Sound good.

**Comment by KrisThielemans** [comment]
thanks @skarrea !

**Pull request merged**